mulog-kafka/test/com/brunobonacci/mulog/publishers/kafka_test.clj

```diff
@@ -0,0 +1,50 @@
+(ns com.brunobonacci.mulog.publishers.kafka-test
+  (:require [midje.sweet :refer :all]
+            [com.brunobonacci.mulog :as mu]
+            [com.brunobonacci.mulog.publishers.container.kafka-setup :as kafka-setup]
+            [clojure.core.async :as async]
+            [jsonista.core :as json]
+            [org.slf4j.impl.mulog]))
```

Comment:
why adding SLF4J bridge here? what's the rationale behind?

Reply:
Debug purposes, the `testcontainers` docker containers output via slf4j.